dbms/CMakeLists.txt

```diff
@@ -312,7 +312,6 @@ if (ENABLE_TESTS)
     )
     target_include_directories(gtests_dbms BEFORE PRIVATE ${SPARCEHASH_INCLUDE_DIR})
     target_compile_definitions(gtests_dbms PUBLIC DBMS_PUBLIC_GTEST)
-    target_compile_definitions(dbms PUBLIC DBMS_PUBLIC_GTEST)
```

Comment:
this break building the bench_xxx
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-build-common/detail/tiflash-build-common/34684/pipeline

Reply:
fixed

`DBMS_PUBLIC_GTEST` should only be set on tests, so should not link to `dbms`